src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(health, name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name, hidden) {
+    super(health, name);
```

Comment:
```suggestion
  constructor() {
    super();
```

you don't actually need those parameters, check it out

Reply:
yep, tests are ok w/o them
but how it's possible even w/o hidden - new properties for this constructor?